plots/charts/chart-loader.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env browser */
+const queryParams = new URLSearchParams(window.location.search);
+const chartParam = queryParams.get('chart') || './grouped-by-metric.js';
+
+const chartScript = document.createElement('script');
```

Comment:
this is a bit clunky, plus the three metrics files are already overlapping pretty hard.

let's merge all of charts/*.js into one file.

you can hook up `generateBloxPlotPerSite()`, or `generateBoxPlotChartPerMetric(); generateLinePlotChartPerMetric();` or `generateGroupedBarChart();` depending on which view we're in.

would rather have a simple slug (e.g. 'by-metric') in the query param or hash instead of a file or function name.